pallets/protos/src/lib.rs

```diff
@@ -90,6 +95,19 @@ pub struct Proto<TAccountId, TBlockNumber> {
 	pub metadata: BTreeMap<Vec<u8>, Hash256>,
 }
 
+#[derive(Encode, Decode, Clone, scale_info::TypeInfo, Debug, PartialEq, Eq)]
+pub struct EthStakeUpdate<TPublic, TBalance> {
```

Comment:
Why is `EthStakeUpdate` defined in pallet_protos and pallet_frag?

Reply:
Mistake, good catch.